test/NameRegistry.upgrade.t.sol

```diff
@@ -190,4 +222,13 @@ contract NameRegistryV2 is
     function setNumber(uint256 _number) public {
         number = _number;
     }
+
+    function supportsInterface(bytes4 interfaceId)
```

Comment:
does this break a test if not included?

Reply:
I believe both AccessControlUpgradeable & ERC721Upgradeable require supportsInterface so I think it needs to be defined here as well

```
error[6480]: TypeError: Derived contract must override function "supportsInterface". Two or more base classes define function with same name and parameter types.
   --> test/NameRegistry.upgrade.t.sol:138:1:
```

Reply:
👍